### DISCUSSION TITLE
Самодельный руль

### DISCUSSION THREAD

**Discussion by lazenyuk-dmitry** [vsulako/AFFBWheel#18] [Show and tell]
Вот я и собрал свой аппарат. Я использовал два мотора 775 с соотношением валов 1:10. Мне хватает с запасом фидбека на 80 процентов.

Вот что собственно получилось.
![IMG_20220703_122232](https://user-images.githubusercontent.com/31993282/177368207-5f9f2d73-2f8b-40bb-82b5-4a802b84edcb.jpg)
![IMG_20220703_122125](https://user-images.githubusercontent.com/31993282/177368226-05010489-ff79-41e9-82e3-0899608c0a8a.jpg)
![IMG_20220703_122117](https://user-images.githubusercontent.com/31993282/177368244-698a0efc-251a-4cac-ac58-a73034622e1b.jpg)
![IMG_20220703_122052](https://user-images.githubusercontent.com/31993282/177368262-769b491c-b93f-40fb-ba02-53f4b9fb5cca.jpg)
![IMG_20220703_122043](https://user-images.githubusercontent.com/31993282/177368287-7cc30bd3-f3ca-4c87-849e-873adfa65804.jpg)
![IMG_20220703_122034](https://user-images.githubusercontent.com/31993282/177368315-04eb4d48-b970-4dfc-b3a1-b6b1ca73f62a.jpg)
![IMG_20220703_121954](https://user-images.githubusercontent.com/31993282/177368358-a078e16f-d180-4103-a7cf-a04435621924.jpg)
![IMG_20220703_121237](https://user-images.githubusercontent.com/31993282/177368382-978c07af-338a-4183-a5dc-3be433a4320f.jpg)
![IMG_20220703_121212](https://user-images.githubusercontent.com/31993282/177368407-5526972f-16fd-4afb-b006-e4f3467b1974.jpg)
![IMG_20220630_103205](https://user-images.githubusercontent.com/31993282/177368435-9e21a2a0-e10b-45c3-96ee-ac1e807331ab.jpg)
![IMG_20220601_022048](https://user-images.githubusercontent.com/31993282/177368453-877b6f7a-c797-476d-a29d-63dc5c0c2d80.jpg)

Ещё доделать по мелочи. Нужно кнопочки на корпус базы поставить. Я сделал 8 кнопок на руль 8 на базу и остальное планирую на КПП вынести и стик тоже. Можно стик разместить и на руль. Установил два вентилятора охлаждения на вдув один и второй на выдув.
Вся конструкция крепится к столу струбцина и с Алика.

@vsulako, как и обещал показываю педали) если будет интересно могу рассказать подробнее и чертежи предоставить. Но профиль мне подогнали. Там уже какой найдешь) поэтому будут чертежи примерными. 

Вообще у меня все это есть в виде 3d модели во freecad. Поэтому вообще могу предоставить модель чертежи, если будет спрос. Могу расписать все что требуется для проекта и ссылки дать на Алик.

Даже сделал лепестки из ЛДСП, фанеры и неодимовых магнитов.
![IMG_20220627_190725](https://user-images.githubusercontent.com/31993282/177370118-a6c5681d-99f3-4095-ab6f-df43489a37cb.jpg)
![IMG_20220627_190717](https://user-images.githubusercontent.com/31993282/177370146-d387a25a-11c5-41e2-a06f-b14e4158732d.jpg)
![IMG_20220627_190707](https://user-images.githubusercontent.com/31993282/177370226-601d3301-229f-453f-8c36-e2df7a7e2f33.jpg)
![IMG_20220627_190701](https://user-images.githubusercontent.com/31993282/177370248-5a1105ab-7d9d-4a46-b443-74e008df3f64.jpg)
![IMG_20220627_190024](https://user-images.githubusercontent.com/31993282/177370281-df68ec25-015b-4cdb-ab4e-8bc254432b65.jpg)
![IMG_20220627_190019](https://user-images.githubusercontent.com/31993282/177370332-dad17400-ac40-49cd-8b82-3ce04ac1dbd8.jpg)
![IMG_20220627_190012](https://user-images.githubusercontent.com/31993282/177370364-39339fe8-5dc3-4fce-89d5-05bfb18456ff.jpg)
![IMG_20220627_190009](https://user-images.githubusercontent.com/31993282/177370382-3b749f16-242e-45a7-977e-6eb46f3d34cb.jpg)

Если кому интересно или будут вопросы можно обсудить. Поделюсь идеями и наработками.

И будет интересно если кто нибудь тоже поделиться своими изобретениями и идеями)

**Comment by vsulako** [comment]
Выглядит круто! 

Ремни натягиваются перемещением моторов?
А как крепится руль к оси? Просто накручивается и затягивается?

А мне то лень, то некогда своим заниматься, все в состоянии вечного прототипа :)

За картинку педалей спасибо, принцип понятен. Интересует только несколько размеров: длина стойки педали,  расстояние меж педалями, и ход педали. И пружины, если они какие-то стандартные и их можно купить.
Мои педали вышли вот такие.

![photo_2022-07-05_20-43-14](https://user-images.githubusercontent.com/95934592/177385658-73b82738-39c2-42fb-a2ec-1001a48c7ee5.jpg)

Оказалось хлипковато, надо было делать каждую на двух стойках или брать профиль раза в два шире. Также оказалось что проблема подыскать пружины, разве что где-то в автозапчастях, но там надо знать что нужно.
Также оказалось неудобно, когда в исходном положении педаль почти вертикальна - надо сильно изгибать ногу. Похоже, было бы удобнее в полулежачем кресле аналогичном автомобильному, но за столом на стуле - не то.
Раздумывал не сделать ли их в обратном виде, как в автомобиле, с осью вращения сверху, но тогда нужна мощная рама. До того руки пока не дошли.

**Reply by lazenyuk-dmitry** [reply]
> Ремни натягиваются перемещением моторов?

Да, совершенно верно.

**Reply by lazenyuk-dmitry** [reply]
> А как крепится руль к оси? Просто накручивается и затягивается?

Ну почти. Жаль я не сделал фото этой конструкции. Я вырезал круг из ЛДСП с отверстием по центру и просто его с двух сторон зажимал гайками на валу, а к нему уже руль. Но это оказалось не надёжно. Прокручивался руль. Тогда я вырезал из нержавейки 1-1.5 мм пластину сделал ее такую же как круг из ЛДСП и приварил гайку по центру, которая накручивается на вал и с двух сторон ещё фиксирую по две гайки. А с другой стороны ЛДСП ещё жестянка такая же но без гайки в центре. и пластины для удобства на саморезы зафиксировал. Вроде надёжно. Такой бутерброд получается в центре ЛДСП с одной стороны жестянка без гайки, а с другой стороны жестянка с приваренной гайкой и все это зажимается гайками с двух сторон на валу.  Если что могу нарисовать.

![IMG_20220706_004302](https://user-images.githubusercontent.com/31993282/177394890-d4354cb3-d527-4343-bdda-16ed66f9b358.jpg)
![IMG_20220706_004251](https://user-images.githubusercontent.com/31993282/177394916-29a1602f-91c2-49bb-8d14-9aae94f2ec0c.jpg)
![IMG_20220706_004238](https://user-images.githubusercontent.com/31993282/177394943-255776f4-a44f-430d-9210-2d42ca6ddfe2.jpg)

Правда для вала пришлось найти стержень на 10 и нарезать резьбы где то 5см с одной стороны. Так как на покупных шпильках резьбы какая то острая и от шпильки на 10 остаётся где то 8мм толщины под резьбой, гнулась потом, ну и лучше на валу 
гладком зафиксировались шестерни. На 10 брал потому что у меня в скользящем контакте отверстие на 10) больше дорого очень.

Вот примерно так:
![IMG_20220701_144236](https://user-images.githubusercontent.com/31993282/177397276-673176db-f13c-4983-925d-a0f446dd13dc.jpg)

Это для педали. Для педалей тоже делал шпильки сам. Не знаю можно ли шпильки найти с нормальной резьбой... В магазине какая то острая трётся неприятно и тормозит. Вроде на алике смотрел есть шпильки нормальные.
![02f9d211756c0843c9c4abeced1e3ace](https://user-images.githubusercontent.com/31993282/177397946-3dbdba7b-8618-472b-af84-0925259ba7d7.jpg)

Кстати руль еле нашёл) один единственный такой в городе на авто рынке. Но вроде есть на Али где то за 8-10 тысяч тенге, с доставкой.

**Reply by lazenyuk-dmitry** [reply]
> Интересует только несколько размеров: длина стойки педали, расстояние меж педалями, и ход педали. И пружины, если они какие-то стандартные и их можно купить.

Постараюсь завтра навести порядок в чертежах и скину размеры. 

Пружины на алике брал две. Завтра скину ссылки. 
На тормоз пружину где то отец нашел в хламе вроде от МТЗ с клапана. Я узнаю ещё точнее. Прям мне нравиться как тормоз ощущается. Правда педали на грани от того чтобы опрокинуться кажется) Но ничего, стоят, я снизу ещё от старого автомобильного коврика четыре куска прикрутил.

**Reply by lazenyuk-dmitry** [reply]
> Оказалось хлипковато, надо было делать каждую на двух стойках или брать профиль раза в два шире.

Профиль мне зять подогнал) он вроде с пласт окна, вроде при монтаже используется, наверно. Тоже узнаю. Профиль был одинакового размера, но если доработать молотком то один в один заходит. Ширина профиля вроде 30мм.

**Reply by lazenyuk-dmitry** [reply]
> Также оказалось неудобно, когда в исходном положении педаль почти вертикальна - надо сильно изгибать ногу.

Ну у меня примерно такой же наклон вроде. Но не замечал неудобства. Сижу на стуле.

**Reply by lazenyuk-dmitry** [reply]
> Раздумывал не сделать ли их в обратном виде, как в автомобиле, с осью вращения сверху, но тогда нужна мощная рама.

Я тоже думал так сделать. Но сделать стенд для педалей. 
https://youtu.be/iC3rP3KujNQ

Есть и заводские такие стойки.

**Reply by lazenyuk-dmitry** [reply]
Я пытался сделать типа как fanatec clubsport v3. Даже хотел сцепление с эффектом корзины сделать, но получилась фигня) Есть идея сделать сцепление как у caravangoes, там другая конструкция но тоже есть эффект корзины на сцеплении.

**Comment by mitiaborodada** [comment]
Пошла жара!
Пора и мне свои наработки выложить.

**Comment by lazenyuk-dmitry** [comment]
@vsulako, вот я расписал https://github.com/lazenyuk-dmitry/SterringWheel если будут вопросы спрашивайте.

**Reply by vsulako** [reply]
Спасибо, очень подробно и наглядно.

**Comment by lazenyuk-dmitry** [comment]
> А как крепится руль к оси? Просто накручивается и затягивается?

@vsulako, привет. Ты интересовался креплением руля. Я тут его переделал. Все же прокручивалось. 

Сделал так.

![IMG20220725141746.jpg](https://user-images.githubusercontent.com/31993282/183295405-465c104c-b230-411a-8dc1-e393f906af4d.jpg)
![IMG20220725125332.jpg](https://user-images.githubusercontent.com/31993282/183295566-39c4b1ef-a5ac-4653-b6fa-ee07109fd0a5.jpg)
![IMG20220725125319.jpg](https://user-images.githubusercontent.com/31993282/183295643-ee510604-8fa9-46c0-971a-9ee74e81cc0e.jpg)
![IMG20220725125307.jpg](https://user-images.githubusercontent.com/31993282/183295664-2581d75d-c037-4689-9172-b8bd65601edd.jpg)
![IMG20220725105453.jpg](https://user-images.githubusercontent.com/31993282/183295681-0ea1a3fd-4eac-487d-8c1c-d7f071548e3e.jpg)
![IMG20220725105442.jpg](https://user-images.githubusercontent.com/31993282/183295691-58eb9922-d79d-4094-91d5-36c2f42178a6.jpg)

Проточил балгаркой гайки как в замок зацеп. Одна была приварена к ступице. Вторую закрутил на вал законтрогаев ее второй гайкой и через гровер ещё и плюс резьбу посадил на фиксатор резьбы (можно купить синий или красный, я использовал китайский супер клей дешёвый) пробовал на откручивание сидят гайки мертво, теперь наверно только греть чтобы открутить). А в гайке на ступице сверлом резьбу срезал, теперь ступица с гайкой просто надевается на стопор типо. Вроде прям мертво все. Подумал может кому будет интересно такое решение.

**Comment by mitiaborodada** [comment]
Всем привет!
Хочу показать немного процесс изготовления своего руля. 
Диаметр руля 320 мм. Сам обод был вырезан из соснового щита. Заготовка оказалась весьма хрупкая в местах, где волокна дерева перпендикулярны касательной окружности обода. Проблему решил просто. выпилил из листа ХДФ такие же 2 обода и прикрутил мелкими саморезами с двух сторон заготовки. В итоге конструкция получилась довольно крепкая. 

Из листа нержавейки 1 мм. вырезал спицы (или как там правильно называется???). 

Обод обернул одним слоем вспененного материала, который используют в качестве подложки под ламинат. После чего обмотал матерчатой изолентой.

Из кожзама сделал оплётку. Сшивал всё вручную толстыми нитками. 

В качестве крепления руля к оси использую держатель вала (такие используют при изготовлении ЧПУ станков) SHF-10

![photo1661692436 (10)](https://user-images.githubusercontent.com/59168456/187077734-949d091b-df2d-4504-9903-5308f21a3c89.jpeg)
![photo1661692436](https://user-images.githubusercontent.com/59168456/187077750-e478b70d-7615-4889-89da-22d912b115d2.jpeg)
![photo1661692436 (8)](https://user-images.githubusercontent.com/59168456/187077737-3622d930-21a5-4a24-995a-124631944c3b.jpeg)
![photo1661692436 (7)](https://user-images.githubusercontent.com/59168456/187077738-54618c2f-0843-46df-8509-a715445d354b.jpeg)
![photo1661692436 (6)](https://user-images.githubusercontent.com/59168456/187077739-175bcc08-f3d0-408b-8f37-00130d113026.jpeg)
![photo1661692436 (5)](https://user-images.githubusercontent.com/59168456/187077740-0c6470d0-8839-4806-8a94-8be5611b0319.jpeg)
![photo1661692436 (9)](https://user-images.githubusercontent.com/59168456/187077735-a2f52863-398c-495b-949f-8818409e449c.jpeg)
![photo1661692436 (4)](https://user-images.githubusercontent.com/59168456/187077742-7759ada8-2681-49b3-8e7c-738ae711f57c.jpeg)
![photo1661692436 (3)](https://user-images.githubusercontent.com/59168456/187077744-ef105674-6899-4799-a85d-1cd8950cccc1.jpeg)
![photo1661692436 (2)](https://user-images.githubusercontent.com/59168456/187077745-18f3d733-9c63-4230-b3f8-bb7b1b72fd77.jpeg)
![photo1661692436 (1)](https://user-images.githubusercontent.com/59168456/187077747-1cd56672-d7f5-4260-9467-d472d945fc0f.jpeg)

**Comment by lazenyuk-dmitry** [comment]
Всем привет. Вот я и сделал все вроде все что можно. 

![IMG20220920150852.jpg](https://user-images.githubusercontent.com/31993282/191217950-338f4491-fc35-4c3d-bfac-d83321036b3e.jpg)
![IMG20220920150824.jpg](https://user-images.githubusercontent.com/31993282/191217994-a0a06ed4-95e6-4360-ab39-81fa1b53885f.jpg)
![IMG20220920150801.jpg](https://user-images.githubusercontent.com/31993282/191218056-1e4740e3-9f0e-45d4-990f-ca8df9aef5ed.jpg)
![IMG20220920150749.jpg](https://user-images.githubusercontent.com/31993282/191218102-4a1627a2-d1fb-4656-a345-cbffbc9c6798.jpg)
![IMG20220920152312.jpg](https://user-images.githubusercontent.com/31993282/191220794-7e41c4c4-b2a2-4d4a-96db-266ea354cea1.jpg)

@vsulako, спасибо огромное за прошивку и ваш труд. Я когда искал подходящею прошивку перерыл много в интернете, уже было думал придется разбираться и писать свою, какие то мелочи то там то там не нравятся, и тут нашел ваш проект на гитхаб. В основном мне в проектах не нравилось как реализованы кнопки. Но теперь у меня 8 кнопок на самой базе, 4 как крестовина слева и 3 для ESC enter и ещё что нибудь. И одна под центровку. 8 кнопок на руле, включая лепестки. 8 передач на КПП в виде кнопок и так же на КПП две просто кнопки два тумблера ВКЛ-ВКЛ - это я думаю если захочется поиграть на грузовиках будет делитель и демультипликатор так же предусмотрел разъем для выноса этих двух тумблеров на ручку КПП, и два тумблера  типа ВКЛ-ВЫКЛ, один стик и одна вертелка потенциометр. Кнопок прям достаточно) 

Василий спасибо за труды и за помощь!

**Reply by vsulako** [reply]
Очень рад :) На вид внушительно.

**Reply by lazenyuk-dmitry** [reply]
@vsulako, кстати вот может быть вам будет интересно. https://github.com/YukMingLaw/ArduinoJoystickWithFFBLibrary попадалась мне такая библиотека типа можно джойстик с фидбэком сделать быстро. Ну если вы не находили эту библиотеку.
